internal/configs/configurator.go

```diff
@@ -1548,6 +1548,42 @@ func (cnf *Configurator) GetIngressCounts() map[string]int {
 	return counters
 }
 
+// GetIngressAnnotations returns a list of annotation keys set across all Ingress resources
+func (cnf *Configurator) GetIngressAnnotations() []string {
+	if cnf == nil || cnf.ingresses == nil {
+		return nil
+	}
+
+	annotationSet := make(map[string]bool)
```

Comment:
I'd say only we add annotations with `nginx` in the set, what do you think @j1m-ryan ?

Reply:
there'll be an exception: `ingress.kubernetes.io/ssl-redirect`

Reply:
and what @oseoin pointed out above

Reply:
This is what I was wondering too, will add these limits so, thank you.